.github/workflows/CI.yml

```
@@ -29,7 +29,6 @@ jobs:
           - "1.6"
           - "1.10"
           - "1"
```

It's not a required check. We can let it fail until it's fixed in the upstream.

```suggestion
          - "1"
          - "nightly"
```